pennylane_lightning_kokkos/src/bindings/Bindings.hpp

```diff
@@ -0,0 +1,137 @@
+// Copyright 2022 Xanadu Quantum Technologies Inc.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+/**
+ * @file Bindings.hpp
+ * Defines operations to export to Python and other utility functions
+ * interfacing with Pybind11
+ */
+#pragma once
+#include "Kokkos_Core.hpp"
+#include <cstring>
+#include <sstream>
+#include <string>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+namespace Pennylane {
+/*@brief Split a string into a string vector based on the delimiters.
+ *
+ *@param str String to be splitted.
+ *@param delimiters Poniter to the delimiters.
+ *@return str_list Substring vector.
+ * */
+auto string_split(const std::string &str, const char *delimiters)
```

Comment:
This and the remaining functions aren't templated header functions, and so don't necessarily belong in a header file in this way. If they are only used in the Bindings.cpp file, best to declare them in there within an anonymous namespace.

Reply:
thanks for the suggestion! done!

Reply:
I don't see these changes here.

Reply:
Still waiting on the changes